### DISCUSSION TITLE
No data for Granada vs Girona - only match missing

### DISCUSSION THREAD

**Discussion by mortgad** [dcaribou/transfermarkt-datasets#203] [Q&A] [answered]
Hi David,

My "customer" noticed that there was one match missing from the latest GW. 
Specifically, it is the game Granada vs Girona that was played this monday (2023-09-18).
I see all other matches played last GW in the file - at least for the Big Five leagues. So it seems a bit strange.

Can you help? 

Thanks, 
Morten

**Comment by dcaribou** [comment, marked as answer]
Hi @mortgad,
This could be due to late arriving data to the Transfermarkt website itself (later than Thursday night anyways, which is when we take the snapshot). I’ve seen this happening occasionally.

Let me trigger a refresh.

**Reply by dcaribou** [reply]
@mortgad - It should be there now.

**Reply by mortgad** [reply]
Yes I see it, awesome! Thanks for the quick reply
